### PR TITLE
Support for Get url command

### PR DESCRIPTION
**Pull request by sandeep-paliwal**
Added command to return Action URLs

## Description

For all actions in manifest
Specific action using 'action' option
CDN URLs for actions using 'cdn' flag

## How Has This Been Tested?

unit tests
CLI run

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [x ] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

File: src/commands/app/get-url.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      if (urls.runtime) {
+        this.log('Runtime URLs')
+        Object.entries(urls.runtime).forEach(([key, value]) => {
+          this.log(chalk.blue(chalk.bold(`${key} `)) + ' - ' + chalk.blue(chalk.bold(`${value} `)))
+        })
+      }
+
+      if (urls.cdn) {
+        this.log('CDN URLs')
+        Object.entries(urls.cdn).forEach(([key, value]) => {
```
Comment (moritzraho) [inline src/commands/app/get-url.js:42]: Question: should we make the log JSON parseable ?

Comment (sandeep-paliwal) [reply src/commands/app/get-url.js:42]: is thing something we do for other commands o/o in general? If yes, then I can modify the PR. Else we can have a separate story to address all command o/p. WDYT?

Comment (moritzraho) [reply src/commands/app/get-url.js:42]: yes ok let's do this as part of another PR.
maybe we could have something like a global --json flag in the future

Comment (sandeep-paliwal) [reply src/commands/app/get-url.js:42]: looks like we have agreement. Resolving the conversation

File: src/commands/app/get-url.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        })
+      }
+      return urls
+    } catch (error) {
+      this.error(wrapError(error))
+    }
+  }
+}
+
+GetUrlCommand.description = 'Get action URL'
```
Comment (moritzraho) [inline src/commands/app/get-url.js:53]: typo: Get action URLs

File: src/commands/app/get-url.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+GetUrlCommand.flags = {
+  ...BaseCommand.flags,
+  cdn: flags.boolean({
+    description: 'Display CDN based action URLs'
+  })
+}
+
+GetUrlCommand.args = [
```
Comment (moritzraho) [inline src/commands/app/get-url.js:62]: Here let's do as in the deploy command and use a multi flag like in the deploy command:
`aio app deploy -a action1 -a action2`
see https://github.com/adobe/aio-cli-plugin-app/blob/master/src/commands/app/deploy.js#L171

File: test/commands/app/geturl.test.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    const retVal = {
+      runtime: {
+        action: 'https://fake_ns.adobeioruntime.net/api/v1/web/sample-app-1.0.0/action'
+      }
+    }
+    mockScripts.getUrls.mockResolvedValue(retVal)
+    const urls = await command.run()
+    expect(command.error).toHaveBeenCalledTimes(0)
+    expect(mockScripts.getUrls).toBeCalledWith({})
+    expect(urls).toBe(retVal)
```
Comment (moritzraho) [inline test/commands/app/geturl.test.js:60]: I think in all test cases we want to also test that the url is logged, as this is the intent of the command.
something like
`expect(command.log).toHaveBeenCalledWith(expect.stringContaining(url))`

File: src/commands/app/get-url.js (hunk 3 above)
Comment (moritzraho) [reply src/commands/app/get-url.js:62]: resolving as this is not in the requirements

File: src/commands/app/get-url.js (hunk 2 above)
Comment (sandeep-paliwal) [reply src/commands/app/get-url.js]: done

File: test/commands/app/geturl.test.js (hunk 4 above)
Comment (sandeep-paliwal) [reply test/commands/app/geturl.test.js:60]: done

**Pull request merged**